include/CMakeLists.txt

```diff
@@ -10,3 +10,13 @@ target_sources(VulkanLayerSettings PRIVATE
 	vulkan/layer/vk_layer_settings.hpp
 	vulkan/layer/vk_layer_settings_ext.h
 )
+
+target_sources(VulkanLayerDispatchTable PRIVATE
+    vulkan/layer/vk_layer_dispatch_table.h
+)
```

Comment:
If we do go with `Vulkan::UtilityHeaders` then we would also add vk_enum_string_helper.h here as well

Reply:
Wait, should there be a target for layer dispatch table, for enum string helper, and one utility headers which is both of those headers?

Reply:
We discussed this in person. But to reiterate. Vulkan::UtilityHeaders would comprise of the dispatch table headers AND enum string helper.